### ISSUE TITLE
At some point, sending a Slack message will fail.

### ISSUE THREAD

**Issue by colondeethpark** [fastlane/fastlane#21258]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

error message -" Error pushing Slack message, maybe the integration has no permission to post on this channel? Try removing the channel parameter in your Fastfile, this is usually caused by a misspelled or changed group/channel name or an expired SLACK_URL "

Slack messages are working fine
At some point it doesn't work.
I looked to see if it was a url problem, but it wasn't a url problem.
Anyone know what could be the cause?

##### Command executed

slack(username: "앱이름",pretext:"<!here>", icon_url:"url", message: ENV["DEPLOYGATE_DEV_MESSAGE"], slack_url: ENV["SLACK_URL"])
##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre> </details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier("com.colondee.simkoong") # The bundle identifier of your app
apple_id("qkralekf@gmail.com") # Your Apple email address

itc_team_id("117167872") # App Store Connect Team ID
team_id("YKC68J5624") # Developer Portal Team ID

# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.212.2 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                                    | Version | Update-Status |
| ----------------------------------------- | ------- | ------------- |
| fastlane-plugin-firebase_app_distribution | 0.5.0   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                                       | Version      |
| ----------------------------------------- | ------------ |
| did_you_mean                              | 1.6.3        |
| rouge                                     | 2.0.7        |
| xcpretty                                  | 0.3.0        |
| terminal-notifier                         | 2.0.0        |
| terminal-table                            | 1.8.0        |
| multipart-post                            | 2.0.0        |
| word_wrap                                 | 1.0.0        |
| optparse                                  | 0.1.1        |
| tty-screen                                | 0.8.1        |
| tty-cursor                                | 0.7.1        |
| tty-spinner                               | 0.9.3        |
| artifactory                               | 3.0.15       |
| babosa                                    | 1.0.4        |
| colored                                   | 1.2          |
| highline                                  | 2.0.3        |
| commander                                 | 4.6.0        |
| faraday-cookie_jar                        | 0.0.7        |
| gh_inspector                              | 1.1.3        |
| naturally                                 | 2.2.1        |
| rubyzip                                   | 2.3.2        |
| security                                  | 0.1.3        |
| xcpretty-travis-formatter                 | 1.0.1        |
| emoji_regex                               | 3.2.3        |
| tsort                                     | 0.1.1        |
| set                                       | 1.0.3        |
| uri                                       | 0.12.0       |
| rexml                                     | 3.2.5        |
| nanaimo                                   | 0.3.0        |
| colored2                                  | 3.1.2        |
| claide                                    | 1.1.0        |
| CFPropertyList                            | 3.0.6        |
| atomos                                    | 0.1.3        |
| xcodeproj                                 | 1.22.0       |
| unicode-display_width                     | 1.8.0        |
| plist                                     | 3.7.0        |
| public_suffix                             | 5.0.1        |
| addressable                               | 2.8.1        |
| excon                                     | 0.99.0       |
| ruby2_keywords                            | 0.0.5        |
| faraday-retry                             | 1.0.3        |
| faraday-rack                              | 1.0.0        |
| faraday-patron                            | 1.0.0        |
| faraday-net_http_persistent               | 1.2.0        |
| faraday-net_http                          | 1.0.1        |
| faraday-multipart                         | 1.0.4        |
| faraday-httpclient                        | 1.0.1        |
| faraday-excon                             | 1.1.0        |
| faraday-em_synchrony                      | 1.0.0        |
| faraday-em_http                           | 1.0.0        |
| faraday                                   | 1.10.0       |
| unf_ext                                   | 0.0.8.2      |
| unf                                       | 0.1.4        |
| domain_name                               | 0.5.20190701 |
| http-cookie                               | 1.0.5        |
| faraday_middleware                        | 1.2.0        |
| fastimage                                 | 2.2.6        |
| json                                      | 2.6.3        |
| mini_magick                               | 4.12.0       |
| dotenv                                    | 2.8.1        |
| bundler                                   | 2.4.7        |
| simctl                                    | 1.6.10       |
| jwt                                       | 2.7.0        |
| webrick                                   | 1.8.1        |
| httpclient                                | 2.8.3        |
| multi_json                                | 1.15.0       |
| signet                                    | 0.17.0       |
| os                                        | 1.1.4        |
| memoist                                   | 0.16.2       |
| googleauth                                | 1.3.0        |
| mini_mime                                 | 1.1.2        |
| retriable                                 | 3.1.2        |
| trailblazer-option                        | 0.1.2        |
| declarative                               | 0.0.20       |
| uber                                      | 0.1.0        |
| representable                             | 3.2.0        |
| google-apis-core                          | 0.11.0       |
| google-apis-playcustomapp_v1              | 0.13.0       |
| google-apis-androidpublisher_v3           | 0.34.0       |
| rake                                      | 13.0.6       |
| digest-crc                                | 0.6.4        |
| google-apis-storage_v1                    | 0.19.0       |
| google-apis-iamcredentials_v1             | 0.17.0       |
| google-cloud-errors                       | 1.3.0        |
| google-cloud-env                          | 1.6.0        |
| google-cloud-core                         | 1.6.0        |
| google-cloud-storage                      | 1.44.0       |
| aws-eventstream                           | 1.2.0        |
| aws-sigv4                                 | 1.5.2        |
| aws-partitions                            | 1.716.0      |
| jmespath                                  | 1.6.2        |
| aws-sdk-core                              | 3.170.0      |
| aws-sdk-kms                               | 1.62.0       |
| aws-sdk-s3                                | 1.119.1      |
| forwardable                               | 1.3.3        |
| logger                                    | 1.5.3        |
| pathname                                  | 0.2.1        |
| shellwords                                | 0.1.0        |
| cgi                                       | 0.3.6        |
| date                                      | 3.3.3        |
| timeout                                   | 0.3.2        |
| stringio                                  | 3.0.5        |
| securerandom                              | 0.2.2        |
| openssl                                   | 3.1.0        |
| digest                                    | 3.1.1        |
| io-nonblock                               | 0.2.0        |
| ipaddr                                    | 1.2.5        |
| io-wait                                   | 0.3.0        |
| zlib                                      | 3.0.0        |
| resolv                                    | 0.2.2        |
| time                                      | 0.2.1        |
| open-uri                                  | 0.3.0        |
| mutex_m                                   | 0.1.2        |
| net-smtp                                  | 0.3.3        |
| net-protocol                              | 0.2.1        |
| net-http-persistent                       | 4.0.1        |
| connection_pool                           | 2.3.0        |
| patron                                    | 0.13.3       |
| ostruct                                   | 0.5.5        |
| english                                   | 0.7.2        |
| erb                                       | 4.0.2        |
| strscan                                   | 3.0.6        |
| abbrev                                    | 0.1.1        |
| io-console                                | 0.6.0        |
| tempfile                                  | 0.1.3        |
| delegate                                  | 0.3.0        |
| tmpdir                                    | 0.1.3        |
| fileutils                                 | 1.7.0        |
| base64                                    | 0.1.1        |
| singleton                                 | 0.1.1        |
| open3                                     | 0.1.2        |
| nkf                                       | 0.1.2        |
| pp                                        | 0.4.0        |
| prettyprint                               | 0.1.1        |
| find                                      | 0.1.1        |
| yaml                                      | 0.2.1        |
| psych                                     | 4.0.3        |
| fastlane-plugin-firebase_app_distribution | 0.5.0        |
</details> </pre>
</details>

### Environment

<details>
  <pre> [REPLACE THIS WITH YOUR INFORMATION] </pre>
</details>

**Comment by Fonceur** [comment]
Yes it's a pain, building 100 iOS targets and it randomly dies, because it can't output to Slack, really?
